From fe75f45fc2732a4b777d1b99250fb0f22db89af7 Mon Sep 17 00:00:00 2001
From: sdhz151 <yanghui140@gmail.com>
Date: Sat, 30 Nov 2024 13:07:44 +0800
Subject: [PATCH] Update Makefile

---
 luci-app-passwall/Makefile | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/package/passwall1/luci-app-passwall/Makefile b/package/passwall1/luci-app-passwall/Makefile
index c87e4d051a..cc0f7b0a81 100644
--- a/package/passwall1/luci-app-passwall/Makefile
+++ b/package/passwall1/luci-app-passwall/Makefile
@@ -11,7 +11,6 @@ PKG_RELEASE:=1
 
 PKG_CONFIG_DEPENDS:= \
 	CONFIG_PACKAGE_$(PKG_NAME)_Iptables_Transparent_Proxy \
-	CONFIG_PACKAGE_$(PKG_NAME)_Nftables_Transparent_Proxy \
 	CONFIG_PACKAGE_$(PKG_NAME)_INCLUDE_Haproxy \
 	CONFIG_PACKAGE_$(PKG_NAME)_INCLUDE_Hysteria \
 	CONFIG_PACKAGE_$(PKG_NAME)_INCLUDE_NaiveProxy \
@@ -55,15 +54,6 @@ config PACKAGE_$(PKG_NAME)_Iptables_Transparent_Proxy
 	select PACKAGE_kmod-ipt-nat
 	default y if ! PACKAGE_firewall4
 
-config PACKAGE_$(PKG_NAME)_Nftables_Transparent_Proxy
-	bool "Nftables Transparent Proxy"
-	select PACKAGE_ipt2socks
-	select PACKAGE_nftables
-	select PACKAGE_kmod-nft-socket
-	select PACKAGE_kmod-nft-tproxy
-	select PACKAGE_kmod-nft-nat
-	default y if PACKAGE_firewall4
-
 config PACKAGE_$(PKG_NAME)_INCLUDE_Haproxy
 	bool "Include Haproxy"
 	select PACKAGE_haproxy
@@ -139,7 +129,7 @@ config PACKAGE_$(PKG_NAME)_INCLUDE_V2ray_Geodata
 	bool "Include V2ray_Geodata"
 	select PACKAGE_v2ray-geoip
 	select PACKAGE_v2ray-geosite
-	default n
+	default y
 
 config PACKAGE_$(PKG_NAME)_INCLUDE_V2ray_Geoview
 	bool "Include V2ray_Geoview"
